app/components-react/pages/RecordingHistory.tsx

```diff
@@ -186,17 +187,19 @@ export function RecordingHistory() {
         )}
       </div>
       <div className={styles.recordingsContainer} id="recordingHistory">
-        {recordings.map(recording => (
-          <div className={styles.recording} key={recording.timestamp}>
-            <span style={{ marginRight: '8px' }}>{formattedTimestamp(recording.timestamp)}</span>
-            <Tooltip title={$t('Show in folder')}>
-              <span onClick={() => showFile(recording.filename)} className={styles.filename}>
-                {recording.filename}
-              </span>
-            </Tooltip>
-            {uploadOptions.length > 0 && <UploadActions filename={recording.filename} />}
-          </div>
-        ))}
+        <Scrollable style={{ height: '100%' }}>
+          {recordings.map(recording => (
+            <div className={styles.recording} key={recording.timestamp}>
+              <span style={{ marginRight: '8px' }}>{formattedTimestamp(recording.timestamp)}</span>
+              <Tooltip title={$t('Show in folder')}>
+                <span onClick={() => showFile(recording.filename)} className={styles.filename}>
+                  {recording.filename}
+                </span>
+              </Tooltip>
+              {uploadOptions.length > 0 && <UploadActions filename={recording.filename} />}
```

Comment:
Nitpick: add parens somewhere assuming prettier doesn't remove it `&& <UploadActions` confused me for a sec 😂

Reply:
i think that was a prettier removal lol